packages/terra-badge/src/Badge.module.scss

```diff
@@ -6,7 +6,7 @@
     border: var(--terra-badge-border);
     border-radius: var(--terra-badge-border-radius, 0.25em);
     display: var(--terra-badge-display, inline-block);
-    font-size: var(--terra-badge-font-size-small, 0.8rem);
+    font-size: var(--terra-badge-small-font-size, 0.8rem);
```

Comment:
I noticed there is another variable below with the same name contained within the small block. Is this variable intended to be identical? Should the default badge be themed separately from the small badge size?

Reply:
yeah, it makes sense to have a separate theme variable for default font size. This could be an error, or maybe it's intended? This is initially set to small here: https://github.com/cerner/terra-core/pull/519

Reply:
updated to default: https://github.com/cerner/terra-core/pull/2125/commits/a3506cef10701a7f7581ac4212d9294c04917a28#diff-efdedb295ee6eff1ea893adcb2918e84R9